guides/common/modules/proc_using-a-custom-repository-on-a-connected-server.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Using_a_Custom_Repository_on_a_Connected_Server_{context}"]
+= Using a Custom Repository on a Connected Server
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} and a disconnected {ProjectServer} via Inter-Satellite Synchronization (ISS) Network Sync.
```

Comment:
+1, probably should be renamed to "Synchronizing a Custom Repository".

Since it synchronizes it on both server I guess there's no need to specify connected/disconnected in the title.